Src/PChecker/CheckerCore/SystematicTesting/TestReport.cs

```diff
@@ -181,7 +181,7 @@ public bool Merge(TestReport testReport)
         /// <summary>
         /// Returns a simple string testing report with only the keys and values.
         /// </summary>
-        public string GetSummaryText(string runtime) {
+        public string GetSummaryText(string runtime, string max_memory) {
```

Comment:
I dont understand why these are passed as string parameters, instead why dont you pass the profiler object and get the information in the function itself?

Reply:
updated to passing in Profiler object in GetSummaryText()